packages/components/pie-modal/README.md

```diff
@@ -0,0 +1,39 @@
+<p align="center">
+  <img align="center" src="../../../readme_image.png" height="200" alt="">
+</p>
+
+<p align="center">
+  <a href="https://www.npmjs.com/@justeattakeaway/pie-modal">
+    <img alt="GitHub Workflow Status" src="https://img.shields.io/npm/v/@justeattakeaway/pie-modal.svg">
+  </a>
+</p>
+
+# pie-modal
+
+This modal is a Web Component built using Lit.
+
+## Local development
+
+Install dependencies at the root
+```
+yarn
+```
+
+Navigate to this folder, compile with TypeScript and build with Vite
+```
+cd packages/components/pie-modal
+yarn build
+```
+
+compile using Vite (auto-compiles `dist` on save)
```

Comment:
I'd argue that everyone knows what "Build and watch" means.

Would leave the technology used out of the readme.

Missing capital letter.

Reply:
```suggestion
Compile and watch for changes (auto-compiles `dist` on save)
```